### ISSUE TITLE
Non-Specific File Description Value & Associated Pin Labels

### ISSUE THREAD

**Issue by hairycactus** [rizonesoft/Firemin#5]
**Observed Behaviour:**
Firemin.exe's _File Description_ value, as well its pins at Start Menu, Taskbar & Notification Area Icons all show "AutoIt Application Framework". See screenshot below.

![Screenshott](https://i.imgur.com/VcgS59p.png)

**Expected Behaviour:**
To avoid confusion, Firemin's _File Description_ value & associated pin labels should show "Firemin", instead of what was used to build Firemin.

**Affected Versions:**
Firemin v6.x (Dec 2017 & Jan 2018), including the latest v6.0.0.4850 (05 Jan 2018)

**Last Known Working Version:**
Firemin v4.0.2.4615 (14 Dec 2016)

**Tested on:** Win 7 x64

Thanks !

**Comment by userOperaFF** [comment]
May no possible change:

https://github.com/rizonesoft/Firemin/blob/5e43ec603b393ad7ae6d28a54fb73710fc2783e7/Firemin.au3#L32

**Comment by hairycactus** [comment]
So fixing [Line 32](https://github.com/rizonesoft/Firemin/blob/5e43ec603b393ad7ae6d28a54fb73710fc2783e7/Firemin.au3#L32)'s `Description field` (currently "`AutoIt Application Framework`") should fix the issue, right ?

**Comment by rizonesoft** [comment]
It is already fixed. Here is a preview on what to expect: [Firemin_4855.zip](https://github.com/rizonesoft/Firemin/files/1635595/Firemin_4855.zip)

**Comment by hairycactus** [comment]
Thanks. I saw & tried v6.0.1.4856 first, followed by v6.0.1.4855 above. I confirm that Firemin is now labelled as Firemin, instead of "AutoIt Application Framework".

The only minor inconsistency is that:
- **File Description value, Taskbar pin, Start Menu pin** -- show "Firemin"
- **Taskbar-System Tray icon** -- shows "Firemin_X64.exe"
- **Notification Area Icons Panel item**
    -- v6.0.1.4855 shows "Firemin_X64.exe" (top row)  | "_Firemin_X64.exe_" (bottom row)
    -- v6.0.1.4856 shows "Firemin" (top row)  | "_Firemin_X64.exe_" (bottom row)

Not sure if the above is making Firemin's system tray icon behave in an unusual manner in v6.0.1.4855 & the just-released v6.0.1.4856. I have opened a [new issue](https://github.com/rizonesoft/Firemin/issues/8) #8  for this.
